mentor_whirlpool/mentor_handles.py

```diff
@@ -75,20 +88,109 @@ async def my_subjects(message):
+
+    if not my_subjects_:
+        my_subjects_ = []
+
+    subjects_to_add = await db.get_subjects()
+
+    for subject in my_subjects_:
+        if subject in my_subjects_:
+            subjects_to_add.remove(subject)
+
+    if subjects_to_add:
+        markup = types.InlineKeyboardMarkup()
+        markup.add(
+            *[types.InlineKeyboardButton(subject, callback_data='sub_add_' + subject) for subject in subjects_to_add])
+        await bot.send_message(call.from_user.id, '<b>Все темы</b>', reply_markup=markup, parse_mode='html')
+    else:
+        await bot.send_message(call.from_user.id, '<b>Вы добавили все доступные темы</b>', parse_mode='html')
+    await bot.answer_callback_query(call.id)
+
+
+@bot.callback_query_handler(func=lambda call: call.data.startswith('sub_add_'))
+async def callback_add_subject(call):
+    db = Database()
+    my_id = (await db.get_mentors(chat_id=call.from_user.id))[0]['id']
+
+    if not (await db.get_mentors(chat_id=call.from_user.id))[0]['subjects'] or call.data[8:] not in \
+            (await db.get_mentors(chat_id=call.from_user.id))[0]['subjects']:
+        await db.add_mentor_subjects(my_id, [call.data[8:]])
+        await bot.send_message(call.from_user.id, f'Тема <b>{call.data[8:]}</b> успешно добавлена', parse_mode='html')
+    else:
+        await bot.send_message(call.from_user.id, f'Тема <b>{call.data[8:]}</b> уже была добавлена', parse_mode='html')
+    await bot.answer_callback_query(call.id)
+
+
+@bot.callback_query_handler(func=lambda call: call.data == 'sub_to_delete')
+async def callback_show_subjects_to_delete(call):
+    db = Database()
+    my_subjects_ = (await db.get_mentors(chat_id=call.from_user.id))[0]['subjects']
+
+    markup = types.InlineKeyboardMarkup()
+    markup.add(
+        *[types.InlineKeyboardButton(subject, callback_data='sub_delete_' + subject) for subject in my_subjects_])
+    await bot.answer_callback_query(call.id)
+    await bot.send_message(call.from_user.id, '<b>Удалить тему</b>', reply_markup=markup, parse_mode='html')
+
+
+@bot.callback_query_handler(func=lambda call: call.data.startswith('sub_delete_'))
+async def callback_add_subject(call):
+    db = Database()
+    my_id = (await db.get_mentors(chat_id=call.from_user.id))[0]['id']
+
+    await db.remove_mentor_subjects(my_id, [call.data[11:]])
+    await bot.answer_callback_query(call.id)
+    await bot.send_message(call.from_user.id, f'Тема <b>{call.data[11:]}</b> успешно удалена', parse_mode='html')
+
+
+@bot.message_handler(func=lambda msg: msg.text == 'Мои студенты')
+async def my_students(message):
+    db = Database()
+
+    mentor_info = (await db.get_mentors(chat_id=message.from_user.id))[0]
+    my_students_ = mentor_info['students']
+
+    if not my_students_:
+        str_my_students_ = 'У Вас нет студентов!'
+    else:
+        str_my_students_ = '\n'.join(
+            '@' + student['name'] + ' - ' + student["course_works"][0]["description"] for student
+            in my_students_)
```

Comment:
Стрёмное форматирование
```suggestion
            '@' + student['name'] + ' - ' + student["course_works"][0]["description"]
            for student in my_students_)
```